unit-test/validate-bump.bats

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ./libs/bats/bin/bats
```

Comment:
Could we move all the stuff related to running tests on your bash script into common/scripts/tests?